sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java

```diff
@@ -14,57 +22,75 @@
-            System.getProperty("ACCOUNT_KEY",
+        properties.getProperty("ACCOUNT_KEY",
                     StringUtils.defaultString(Strings.emptyToNull(
                             System.getenv().get("ACCOUNT_KEY")),
                             "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="));
 
     public static String SECONDARY_MASTER_KEY =
-        System.getProperty("SECONDARY_ACCOUNT_KEY",
+        properties.getProperty("SECONDARY_ACCOUNT_KEY",
             StringUtils.defaultString(Strings.emptyToNull(
                 System.getenv().get("SECONDARY_ACCOUNT_KEY")),
                 "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="));
 
     public static String HOST =
-            System.getProperty("ACCOUNT_HOST",
+        properties.getProperty("ACCOUNT_HOST",
                     StringUtils.defaultString(Strings.emptyToNull(
                             System.getenv().get("ACCOUNT_HOST")),
                             "https://localhost:443/"));
 
     public static String CONSISTENCY =
-            System.getProperty("ACCOUNT_CONSISTENCY",
+        properties.getProperty("ACCOUNT_CONSISTENCY",
                                StringUtils.defaultString(Strings.emptyToNull(
                                        System.getenv().get("ACCOUNT_CONSISTENCY")), "Strong"));
 
     public static String PREFERRED_LOCATIONS =
-            System.getProperty("PREFERRED_LOCATIONS",
+        properties.getProperty("PREFERRED_LOCATIONS",
                                StringUtils.defaultString(Strings.emptyToNull(
                                        System.getenv().get("PREFERRED_LOCATIONS")), null));
 
     public static String MAX_RETRY_LIMIT =
-            System.getProperty("MAX_RETRY_LIMIT",
+        properties.getProperty("MAX_RETRY_LIMIT",
                                StringUtils.defaultString(Strings.emptyToNull(
                                        System.getenv().get("MAX_RETRY_LIMIT")),
                                                          "2"));
 
     public static String DESIRED_CONSISTENCIES =
-            System.getProperty("DESIRED_CONSISTENCIES",
+        properties.getProperty("DESIRED_CONSISTENCIES",
                                StringUtils.defaultString(Strings.emptyToNull(
                                        System.getenv().get("DESIRED_CONSISTENCIES")),
                                                          null));
 
     public static String PROTOCOLS =
-            System.getProperty("PROTOCOLS",
+        properties.getProperty("PROTOCOLS",
                                StringUtils.defaultString(Strings.emptyToNull(
                                        System.getenv().get("PROTOCOLS")),
                                                          null));
+
+    private static Properties loadProperties() {
+        Path path = Paths.get(System.getProperty("user.home"), "cosmos-v4.properties");
```

Comment:
How about having default file cosmos-v4.properties in project directory itself , so dev just need to change that instead of creating new in local

Reply:
good suggestion Naveen.
I added the support for that as well.

Reply:
done